README.md

```diff
@@ -1,31 +1,7 @@
 # Gnosis Safe Android App [![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
 
-**WARNING: Under development. Don't use the application with real funds! Application right now targets the Rinkeby test network. Switching to mainnet (or any other ethereum network) can be done by the user but it's its responsibility in doing so.**
-
 ## Goal
-The goal of this project is to provide a 2FA-enabled Ethereum Wallet. This can be achieved by using the new [Gnosis Safe](https://github.com/gnosis/gnosis-safe-contracts) (the new version of our [Multisig Wallet](https://github.com/gnosis/MultiSigWallet)). 
-
-You can add multiple devices to the safe and secure your wallet by requiring a set of devices to confirm each transaction. Depending on you security setting and number of devices you can still access your funds even if you loose access to a device.
-
-Deploying and interacting with your Gnosis Safe will also be much cheaper than with our Multisig Wallet.
-
-## Roadmap
-### Current Features
-* Deploying and Restoring Safes
-* Creating and restoring account access (via mnemonic phrase)
-* Transfer funds (ether or tokens)
-* Address Book (so you don't need to memorize all addresses)
-* Manage Tokens - verify token addresses that you trust so you can access their information easily
-* Mnemonic reveal for active account
-* Fiat prices for transaction costs (USD)
-* Fingerprint support
-* Add/Remove owner from deployed safes
-
-### To be added
-* Trezor support
-* Transaction scanning with QRCode
-* Integration with external protocols/apps for signing
-
+The goal of this project is to provide a 2FA-enabled Ethereum Wallet. This can be achieved by using the new [Gnosis Safe](https://github.com/gnosis/safe-contracts) (the new version of our [Multisig Wallet](https://github.com/gnosis/MultiSigWallet)).
```

Comment:
I know it's a page for technical users, but I'd reuse some of the things we already have. How about the following text:
```
The Gnosis Safe aims to provide all users with a convenient, yet secure way to manage their funds and interact with decentralized applications on Ethereum. 
Transactions are secure by on-chain 2-factor-authentication. This is achieved by using the [Gnosis Safe smart contracts](https://github.com/gnosis/safe-contracts). They are the successor of the broadly trusted [Gnosis Multisig Wallet smart contracts](https://github.com/gnosis/MultiSigWallet)).
```